src/core/helpers/http-status.ts

```diff
@@ -1,3 +1,26 @@
+export const OK = 200;
+export const CREATED = 201;
+export const ACCEPTED = 202;
+export const NO_CONTENT = 204;
+export const MOVED_PERMANENETLY = 301;
+export const SEE_OTHER = 303;
+export const NOT_MODIFIED = 304;
+export const TEMPORARY_REDIRECT = 307;
+export const INVALID_JSON = 400;
+export const AUTHORIZATION_FAILURE = 401;
```

Comment:
This is redundant to `UNAUTHORIZED` -- keep `UNAUTHORIZED` and get rid of `AUTHORIZATION_FAILURE`

Reply:
Let's actually get rid of this and just make it `UNAUTHORIZED`

Reply:
addressed.